spec/unit/routing/router_spec.rb

```diff
@@ -0,0 +1,89 @@
+require "spec_helper"
```

Comment:
**CodeQuality - Nitpick**
Add unit specs for router in addition to integration specs.